core/src/com/unciv/models/ruleset/Era.kt

```diff
@@ -46,3 +75,33 @@ class Era : INamed, IHasUniques {
     /** This is used for display purposes in templates */ 
     override fun toString() = name
 }
+
+class CityStateBonus(val text:String) {
```

Comment:
Why do we need another class for this, this is just Unique again
And if you want enum recognition, there's a new method for that as well with UniqueType